pages/recipe-details.vue

```diff
@@ -36,10 +37,8 @@
       </div>
       <button @click="addToShoppingList">Add to shopping list</button>
       <h2 class="recipe-title">Steps</h2>
-      <div v-for="step in this.recipeData.steps" :key="step">
-        <ol>
-          <li>{{ step }}</li>
-        </ol>
+      <div v-for="(step, index) in this.recipeData.steps" :key="step">
+        <div class="margins">{{ index + 1 }}. {{ step }}</div>
```

Comment:
The margins class is currently in index.vue and not working on the staging server. Can you move this style to default.vue?